src/index.js

```diff
@@ -22,11 +22,14 @@ export const SessionToken: Token<Session> = createToken('SessionToken');
 
 export type Logger = {
   log(level: string, arg: any): void,
-  error(arg: any): void,
-  warn(arg: any): void,
-  info(arg: any): void,
-  verbose(arg: any): void,
-  debug(arg: any): void,
-  silly(arg: any): void,
+  error(arg: any, error: any): void,
+  warn(arg: any, arg: any): void,
+  info(arg: any, arg: any): void,
+  verbose(arg: any, arg: any): void,
+  debug(arg: any, arg: any): void,
+  silly(arg: any, arg: any): void,
+  +access?: (arg: any, arg: any) => void,
```

Comment:
These are new methods to support some additional levels that we have internally. Seemed like they should be new, optional values?

Reply:
Are these specific to `Logtron`?  If so, this feels a little odd to have a generic logger interface include implementation-specific methods.

For comparison, the `Winston` logger does not include these:
```
interface LeveledLogMethod {
    (msg: string, callback: LogCallback): LoggerInstance;
    (msg: string, meta: any, callback: LogCallback): LoggerInstance;
    (msg: string, ...meta: any[]): LoggerInstance;
}

interface LogMethod {
    (level: string, msg: string, callback: LogCallback): LoggerInstance;
    (level: string, msg: string, meta: any, callback: LogCallback): LoggerInstance;
    (level: string, msg: string, ...meta: any[]): LoggerInstance;
}

class Winston {
  ...

  log: LogMethod;

  silly: LeveledLogMethod;
  debug: LeveledLogMethod;
  verbose: LeveledLogMethod;
  info: LeveledLogMethod;
  warn: LeveledLogMethod;
  error: LeveledLogMethod;
  ...
}
```
[main/types/winston/index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/winston/index.d.ts)

Alternatively, see the [Flow definitions](https://github.com/flowtype/flow-typed/blob/master/definitions/npm/winston_v3.x.x/flow_v0.34.x-/winston_v3.x.x.js#L50) (which are slightly less clear).

Reply:
Yes, makes sense, but it seems that some folks may rely and use these methods. Perhaps we need a separate token if we have different method?